R/params.R

```diff
@@ -186,6 +186,56 @@ params_namedList <- function() {
   empty
 }
 
+
+setup_html_head <- function(html_head_style = c(),
```

Comment:
nit: all of our other param helper functions have "params" in their name. It's not the best namespacing, but it's something.